.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+    extends: [
+        "@khanacademy",
+    ],
+    plugins: ["import", "jest", "promise", "monorepo"],
+    settings: {
+        react: {
+            version: "detect",
+        },
+    },
+    overrides: [
+        {
+            files: ["utils/*.js"],
+            rules: {
+                "import/no-commonjs": "off",
+            },
+        },
+    ],
+    rules: {
+        "flowtype/require-exact-type": ["error", "always"],
+        "flowtype/no-types-missing-file-annotation": "error",
+        "import/no-unresolved": "error",
+        "import/named": "error",
+        "import/default": "error",
+        "import/no-absolute-path": "error",
+        "import/no-self-import": "error",
+        "import/no-useless-path-segments": "error",
+        "import/no-named-as-default": "error",
+        "import/no-named-as-default-member": "error",
+        "import/no-deprecated": "error",
+        "import/no-commonjs": "error",
+        "import/first": "error",
+        "import/no-duplicates": "error",
+        "import/order": [
+            "error",
+            {
+                groups: ["builtin", "external"],
+            },
+        ],
+        "import/newline-after-import": "error",
+        "import/no-unassigned-import": "error",
+        "import/no-named-default": "error",
+        "import/extensions": [
+            "error",
+            "always",
+            {
+                ignorePackages: true,
+            },
+        ],
+        "jest/no-focused-tests": "error",
+        "promise/always-return": "error",
+        "promise/no-return-wrap": "error",
+        "promise/param-names": "error",
+        "promise/catch-or-return": "error",
+        "promise/no-new-statics": "error",
+        "promise/no-return-in-finally": "error",
+        "monorepo/no-internal-import": "error",
+        "monorepo/no-relative-import": "error",
+        "import/no-restricted-paths": [
+            "error",
+            {
+                zones: [
+                    {
+                        target: "./packages/(?!.*test.js).*",
+                        from: "utils",
+                    },
+                ],
```

Comment:
If I'm reading this correctly, we're disallowing importing from the `utils` directory from all packages but excluding test files.  If that's the case, looks good to me.